rpg.py

```diff
@@ -41,56 +54,136 @@ def deal_damage(self, other_player):
         return other_player
 
     def get_all_enemies(self, players: List) -> List[int]:
+        """returns the location of all enemy players in a list"""
         return [
             index for index, player in enumerate(players)
             if not self.is_ally(player) and player.is_alive()
         ]
 
+    def get_all_allies(self, players: List) -> List[int]:
+        """returns the locations of all ally players in a list"""
+        return[
+            index for index, player in enumerate(players)
+            if self.is_ally(player1) and self.is_alive()
```

Comment:
`player1`? Shouldn't it be `player`?

Reply:
player1 is the object of the user as the word player has been used too many different times and it confused me.